### ISSUE TITLE
Setup window scaling is bigger than expected.

### ISSUE THREAD

**Issue by Segilmez06** [ollama/ollama#8160]
### What is the issue?

It's probably not causing any problems during usage but this kinda bothers me on each update. The setup window, text and buttons are larger than expected. I'm currently on `0.5.4` but same exists on previus versions. I have dual monitor setup. Both 1080p and never used scaling (on default %100). The size difference is pretty obvious in eye but here is a screenshot:

![image](https://github.com/user-attachments/assets/0b2913c4-87af-4465-bb89-c384cfa15e07)
Right window for reference. All setup programs I have ever seen is in size of it.

### OS

Windows

### GPU

Nvidia

### CPU

Intel

### Ollama version

0.5.4

**Comment by YonTracks** [comment]
I managed to have some luck with ollama.iss/ ```WizardSizePercent=100``` and lowered the font size ```[LangOptions]
DialogFontSize=10``` huge difference. cheers!
same same bugged me lol.
maybe the maintainers will see and agree.
good luck

**Comment by Segilmez06** [comment]
> I managed to have some luck with ollama.iss/ `WizardSizePercent=100` and lowered the font size `[LangOptions] DialogFontSize=10` huge difference. cheers! same same bugged me lol. maybe the maintainers will see and agree. good luck

I actually don't think maintainers would see this issue among other 1.1K open. I haven't tested yet but if those worked, a PR would be better in my opinion.

**Comment by YonTracks** [comment]
yep, I agree, I meant contributors rather anybody, if it is good enough. seems like maybe a good first ever PR. 
![Screenshot 2024-12-21 085856](https://github.com/user-attachments/assets/9b92480e-a3e0-4679-9dbe-116ef1553de9)
![Screenshot 2024-12-21 090033](https://github.com/user-attachments/assets/7a303085-b30a-48a8-bad6-d1c6e48c2cc8)
still not quite but might be good enough.

**Comment by YonTracks** [comment]
> > I managed to have some luck with ollama.iss/ `WizardSizePercent=100` and lowered the font size `[LangOptions] DialogFontSize=10` huge difference. cheers! same same bugged me lol. maybe the maintainers will see and agree. good luck
> 
> I actually don't think maintainers would see this issue among other 1.1K open. I haven't tested yet but if those worked, a PR would be better in my opinion.

I did try a PR, not enough skills lol, github wants to create a PR for more than just this from my repo? I think I need to make just the 1 repo change, then commit the change, then PR. I will see how I go.
cheers. might take me a bit lol.  feel free to do it.

**Comment by Segilmez06** [comment]
Great! I hope this gets merged before next release.
